src/Parser/CommandParser.php

```diff
@@ -5,16 +5,20 @@
 namespace Synolia\SyliusSchedulerCommandPlugin\Parser;
 
 use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
 use Symfony\Component\HttpKernel\KernelInterface;
 use Webmozart\Assert\Assert;
 
 class CommandParser implements CommandParserInterface
 {
     /** @var string[] */
-    private array $excludedNamespaces;
+    private readonly array $excludedNamespaces;
 
-    public function __construct(private KernelInterface $kernel, array $excludedNamespaces = [])
-    {
+    public function __construct(
+        private readonly KernelInterface $kernel,
+        #[Autowire(['_complete'])]
```

Comment:
I don't see any namespace that could match with this one. I saw it was set in services.yaml, but really weird

Reply:
:shrug: